test/task_card.test.js

```diff
@@ -1,7 +1,7 @@
 const Application = require('spectron').Application;
 const assert = require('assert');
 
-describe('Application launch', function () {
+describe('Spectron tests', function () {
     this.timeout(10000);
```

Comment:
Is this timeout required?

Reply:
I don't think so. Removing it should be okay.

Reply:
Actually, this might be necessary for Travis.